webapp/channels/src/components/channel_layout/channel_controller.tsx

```diff
@@ -51,11 +51,13 @@ export default function ChannelController(props: Props) {
         };
     }, []);
 
+    // Starts a regular interval to fetch statuses of users.
+    // see function "addVisibleUsersInCurrentChannelAndSelfToStatusPoll" for more details on which user's statuses are fetched.
     useEffect(() => {
         let loadStatusesIntervalId: NodeJS.Timeout;
         if (enabledUserStatuses) {
             loadStatusesIntervalId = setInterval(() => {
-                dispatch(addVisibleUsersInCurrentChannelToStatusPoll());
+                dispatch(addVisibleUsersInCurrentChannelAndSelfToStatusPoll());
```

Comment:
As per your comment in MM, did you want to also rename "Poll" to "Pool" here?

Reply:
No i thought about it a bit more term "polling" refers to the process of periodically querying or fetching data from a source.